modules/ROOT/pages/database-configuration.adoc

```diff
@@ -72,7 +74,7 @@ Also, note that the owner of the new schemas must own the following privileges:
 [NOTE]
 ====
 
-CREATE TABLE and CREATE INDEX privileges are not required after first start in normal use. +
+CREATE TABLE and CREATE INDEX privileges are not required for the *engine database* user after first start in normal use (however, they will still be required by the *business data database* user even beyond the first deployment of a BDM). +
```

Comment:
We cannot say that the owners of both databases must have the same privileges and then say that some privileges are not required for one of them.
For more clarity, I would suggest to first describe the privileges of the Bonita Engine database owner, and then the privileges of the business data database.

Reply:
And instead of **Engine**, I would say **Runtime**

Reply:
* I agree with @NathalieC , this note is not useful and there are too many conditions for these priviledges aren't required. But 90% of the time, they are required (except when "first start in normal use" which is not clear to me, please define "first start in normal use"...). Let's say they are always required so we cover all the use cases. And we can remove that note. 
* Regarding the Engine vs Runtime, is that the new word to be used in the documentation? If that is the case, then we need to review all the pages that talk about the Engine. I say Engine is fine for me.

Reply:
@mariellebonita Fine to me.
I let you do this last removal and merge the PR like that (the rest is fine to me).

Reply:
@mariellebonita I removed the note